.github/workflows/upload-docs.yml

```diff
@@ -0,0 +1,54 @@
+name: Build and upload API docs
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to publish docs under (e.g. `v1.2.3-dev.1`)'
+        required: true
+
+env:
+  GH_TOKEN: ${{ github.token }}
+
+permissions:
+  actions: 'write'
+
+jobs:
+  build-wasm: 
+    # owner/repository of workflow has to be static, see https://github.community/t/env-variables-in-uses/17466
+    uses: iotaledger/identity.rs/.github/workflows/shared-build-wasm.yml@main
+    with:
+      run-unit-tests: false
+      ref: ${{ inputs.ref }}
+      output-artifact-name: identity-docs
+
+  upload-docs:
+    runs-on: ubuntu-latest
+    needs: build-wasm
+    steps:
+      - uses: actions/download-artifact@v2
```

Comment:
Should also be updated to v4. As even v3 will be deprecated this year. But we can do that in another PR too

Reply:
fine to update it here

Reply:
4731a6fc1e9dfed981f078295b729d77b7b329a8